src/less/styles.less

```diff
@@ -45,7 +45,7 @@
 
 html {
   color: @color__text;
-  font: @font__size sans-serif;
+  font: @font__size 'Open Sans';
```

Comment:
Shouldn't it be `'Open Sans', sans-serif` instead? Custom fonts might be disabled and browser needs to have a legit fallback